stress-example/image_search/app.py

```diff
@@ -36,11 +36,11 @@ def random_docs(start, end):
 
 
 def config(indexer_query_type):
-    parallel = 1
-    shards = 2
+    shards_encoder = 1
+    shards_indexers = 2
 
-    os.environ['JINA_PARALLEL'] = str(parallel)
-    os.environ['JINA_SHARDS'] = str(shards)
+    os.environ['JINA_SHARDS_ENCODER'] = str(shards_encoder)
+    os.environ['JINA_SHARDS_INDEXER'] = str(shards_indexers)
```

Comment:
`INDEXERS`, to make it clear it's used for both (and to make it consistent with my changes)

Reply:
Done